nvflare/private/fed/server/job_cmds.py

```diff
@@ -210,26 +211,34 @@ def _job_match(job_meta: Dict, id_prefix: str, name_prefix: str) -> bool:
     @staticmethod
     def _send_detail_list(conn: Connection, jobs: List[Job]):
         for job in jobs:
+            JobCommandModule._set_duration(job)
             conn.append_string(json.dumps(job.meta, indent=4))
 
     @staticmethod
     def _send_summary_list(conn: Connection, jobs: List[Job]):
 
-        table = Table(["Job ID", "Name", "Status", "Submit Time"])
+        table = Table(["Job ID", "Name", "Status", "Submit Time", "Run Duration"])
         for job in jobs:
+            JobCommandModule._set_duration(job)
             table.add_row(
                 [
                     job.meta.get(JobMetaKey.JOB_ID, ""),
                     CommandUtil.get_job_name(job.meta),
                     job.meta.get(JobMetaKey.STATUS, ""),
                     job.meta.get(JobMetaKey.SUBMIT_TIME_ISO, ""),
+                    str(job.meta.get(JobMetaKey.DURATION, "N/A")),
```

Comment:
The output will be more readable if converted to datetime.timedelta

Reply:
Addressed.